OpenRA.Mods.Common/Traits/Parachutable.cs

```diff
@@ -71,7 +72,8 @@ public void OnLanded()
 			if (sequence != null && palette != null)
 				self.World.AddFrameEndTask(w => w.Add(new Explosion(w, self.OccupiesSpace.CenterPosition, sequence, palette)));
 
-			self.Kill(self);
+			if (self.World.ActorMap.GetActorsAt(self.Location).Any(a => a != ignore))
```

Comment:
What are you trying to achieve with this? That would probably help. Right now I would say the interface change isn't even needed.